generators/app/templates/src/utils/notification.js

```diff
@@ -0,0 +1,37 @@
+import { ldflexHelper } from './index';
+
+export const sendNotification = async (opponent, content, createNotification, to) => {
+  try {
+    if (to) {
+      return createNotification(content, to);
+    }
+    /**
+     * If the opponent doesn't has inbox we show an error
+     */
+    throw new Error('Error the opponent does not has inbox to send notification');
```

Comment:
Should read:

"Error: The opponent doesn't have an available inbox"

Also this should be internationalized